src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -51,93 +47,95 @@
     :description       :text
     :description-props {:text (string/replace derivation-path #"/" " / ")}}])
 
-(defn- view-internal
+(defn- f-view
   []
   (let [top                   (safe-area/get-top)
         bottom                (safe-area/get-bottom)
         account-color         (reagent/atom (rand-nth colors/account-colors))
         emoji                 (reagent/atom (emoji-picker.utils/random-emoji))
-        number-of-accounts    (count (rf/sub [:wallet/accounts-without-watched-accounts]))
-        account-name          (reagent/atom "")
-        placeholder           (i18n/label :t/default-account-placeholder
-                                          {:number (inc number-of-accounts)})
+        number-of-accounts    (count (rf/sub [:wallet/accounts]))
+        account-name          (reagent/atom (i18n/label :t/default-account-placeholder
+                                                        {:number (inc number-of-accounts)}))
```

Comment:
I think you need to be careful with the rebase here as it looks like you are removing the changes added here: https://github.com/status-im/status-mobile/pull/18683/files#diff-f5c9367f2785be0d7ef5d979f937a88bebd69956e1d1bb1f1f41d09201df8b78

Reply:
@J-Son89 thank you for noticing this, fixed 👍